repository-data/application/src/main/resources/hcm-config/configuration/translations/cms.yaml

```diff
@@ -66,6 +66,7 @@ definitions:
         ${supportedEmbedSrcPrefixes}'
       hee:media-embed-code-validator#invalid-embed-code-error: Enter a valid embed
         code
+      hee:mandatory-publication-type-validator: Please select a Publication type
```

Comment:
Could we remove the unused translation `hee:mandatory-publication-type-validator` ?

Reply:
Will do